ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -121,299 +121,300 @@ Rectangle {
+            if (root.isPinned)
+                return Theme.palette.pinColor3;
+            return "transparent";
+        }
     }
 
-    Rectangle {
-        anchors {
-            top: parent.top
-            bottom: parent.bottom
-            left: parent.left
+    contentItem: Item {
+        Rectangle {
+            anchors {
+                top: parent.top
+                bottom: parent.bottom
+                left: parent.left
+            }
+            width: 2
+            visible: root.isPinned
+            color: Theme.palette.pinColor1
         }
-        width: 2
-        visible: root.isPinned
-        color: Theme.palette.pinColor1
-    }
 
-    Rectangle {
-        anchors {
-            top: parent.top
-            bottom: parent.bottom
-            left: parent.left
+        Rectangle {
+            anchors {
+                top: parent.top
+                bottom: parent.bottom
+                left: parent.left
+            }
+            width: 2
+            visible: root.hasMention
+            color: Theme.palette.mentionColor1
         }
-        width: 2
-        visible: root.hasMention
-        color: Theme.palette.mentionColor1
-    }
 
-    SequentialAnimation {
-        id: messageFoundAnimation
+        SequentialAnimation {
+            id: messageFoundAnimation
 
-        PauseAnimation {
-            duration: 600
-        }
-        NumberAnimation {
-            target: highlightRect
-            property: "opacity"
-            to: 1.0
-            duration: 1500
-        }
-        PauseAnimation {
-            duration: 1000
-        }
-        NumberAnimation {
-            target: highlightRect
-            property: "opacity"
-            to: 0.0
-            duration: 1500
+            PauseAnimation {
+                duration: 600
+            }
+            NumberAnimation {
+                target: highlightRect
+                property: "opacity"
+                to: 1.0
+                duration: 1500
+            }
+            PauseAnimation {
+                duration: 1000
+            }
+            NumberAnimation {
+                target: highlightRect
+                property: "opacity"
+                to: 0.0
+                duration: 1500
+            }
         }
-    }
 
-    Rectangle {
-        id: highlightRect
-        anchors.fill: parent
-        opacity: 0
-        visible: opacity > 0.001
-        color: Theme.palette.baseColor2
-    }
-
-    MouseArea {
-        id: mouseArea
-        anchors.fill: parent
-    }
+        Rectangle {
+            id: highlightRect
+            anchors.fill: parent
+            opacity: 0
+            visible: opacity > 0.001
+            color: Theme.palette.baseColor2
+        }
 
-    HoverHandler {
-        id: hoverHandler
-        enabled: !root.isActiveMessage && !root.disableHover
-    }
+        MouseArea {
+            id: mouseArea
+            anchors.fill: parent
+        }
```

Comment:
And in the same manner, I wonder what's the purpose of this MouseArea, it doesn't seem to be used anywhere inside this component?

Reply:
it's an alias (line 31) and used in MessageView.qml. besides enabled and onClicked that are provided by the Control we need to also set acceptedButtons. any ideas if not with MouseArea?

Reply:
Hmm... not :/